sql-scripts/functions.sql

```diff
@@ -2485,7 +2485,7 @@ FOR v_branch IN
 					WHERE 	id = v_branch.f_bus OR 
 						id = v_branch.t_bus);	
 					
-		v_numb_transformers := (SELECT ceil(v_S_long_MVA_sum_min / (SELECT S_MVA FROM transformer_specifications WHERE U_OS = v_U_OS AND U_US = v_U_US))); -- Wird auf nächste ganze Zahl aufgerundet.
+		v_numb_transformers := (SELECT CASE WHEN ceil(v_S_long_MVA_sum_min / (SELECT S_MVA FROM transformer_specifications WHERE U_OS = v_U_OS AND U_US = v_U_US)) = 0 THEN 1 ELSE ceil(v_S_long_MVA_sum_min / (SELECT S_MVA FROM transformer_specifications WHERE U_OS = v_U_OS AND U_US = v_U_US)) END); -- Wird auf nächste ganze Zahl aufgerundet.
```

Comment:
Does that mean that the s_nom is set to 1 in case no capacity is added to one site of the transformer?

Reply:
I am asking because 
`ceil(v_S_long_MVA_sum_min / (SELECT S_MVA FROM transformer_specifications WHERE U_OS = v_U_OS AND U_US = v_U_US))` 
would return 1 even if 
`(v_S_long_MVA_sum_min / (SELECT S_MVA FROM transformer_specifications WHERE U_OS = v_U_OS AND U_US = v_U_US))` 
is very small but not zero.

Reply:
the minimum number of transformers should be 1 and not 0. Otherwise it creates problems in the following - and I think it is ok to assume at least one transformer per substation (even though there might be in some weird cases v_S_long_MVA_sum_min = 0). Do you agree?

Reply:
I had a look how many substations would be effected - it looks like there are two HV/eHV substations/trafos where no 110kV line is attached to, but only eHV lines. 
So there are at least only few cases like this. And yes, I agree that it would be worse to not have any transformer there at all. 
Should I check if the updated SQL query works or did you already do that?

Reply:
I tried it and it worked. The minimum number is now 1